src/BalanceTx.purs

```diff
@@ -240,6 +244,13 @@ derive instance genericBalanceNonAdaOutsError ::
 instance showBalanceNonAdaOutsError :: Show BalanceNonAdaOutsError where
   show = genericShow
 
+data TxInputLockedError = TxInputLockedError
+
+derive instance genericTxInputLockedError :: Generic TxInputLockedError _
```

Comment:
I know you were just following the convention in this module, but we actually got rid of all instance names in the rest of the codebase (they're optional now and in most cases just add visual noise). Since you're already modifying this module a bit, could you remove these names as well as any others you find please?

Reply:
Of course, done.